autogpt_platform/frontend/package.json

```diff
@@ -10,14 +10,14 @@
     "start:standalone": "cd .next/standalone && node server.js",
     "lint": "next lint && prettier --check .",
     "format": "prettier --write .",
-    "type-check": "tsc --noEmit",
+    "types": "tsc --noEmit",
     "test": "next build --turbo && playwright test",
-    "test-ui": "next build --turbo && playwright test --ui",
-    "test:no-build": "playwright test",
+    "test:ui": "next build --turbo && playwright test --ui",
+    "test:unit": "vitest --config vitest.config.mjs --run",
+    "test:unit:watch": "vitest --config vitest.config.mjs --watch",
```

Comment:
why swap from - to :?

Reply:
Because we have both mixed `-` or `:` . I don't mind one or the other but is better to stay consistent on a pattern.

So either:
```
test-ui
test-unit
test-unit-watch
```
or 
```
test:ui
test:unit
test:unit-watch
```

I'm more used to `:` but happy to change to `-` too 🤷🏽

Reply:
okay cool just curious